### ISSUE TITLE
Moving towards resources management (execution domain)

### ISSUE THREAD

**Issue by fred-r** [Open-CMSIS-Pack/Open-CMSIS-Pack-Spec#205]
Some time back we started to discuss the possibility to integrate the rzone concept.
Can we start discussing it ?

Typically, the current rzone format probably needs to be reworked a bit with regards to some items like DnumMpuRegions as the MPU is banked on Cortex-M33.

**Comment by ReinhardKeil** [comment]
Covered now by https://open-cmsis-pack.github.io/cmsis-toolbox/Experimental-Features/#resource-management - should be closed once this is complete.

**Comment by iomint** [comment]
[After Open-CMSIS-pack weekly call held 25w07, when Open-CMSIS-Pack stakeholders were asked to provide feedback]

Reinhard's proposal in [https://open-cmsis-pack.github.io/cmsis-toolbox/Experimental-Features/#resource-management](https://open-cmsis-pack.github.io/cmsis-toolbox/Experimental-Features/#resource-management) addresses the point raised in this ticket so "technically" the "wontfix" label seems no more true. 
Concerning the content, this new resource management proposal seems interesting, we are going to review and discussed it internally in the coming few weeks.
